en/docs/try-out/account-and-transaction-flow.md

```diff
@@ -13,55 +14,68 @@ This document provides step by step instructions to deploy, subscribe, and invok
 
 3. Select **OpenAPI File/Archive**. ![create-an-api](../assets/img/get-started/quick-start-guide/create-an-api.png)
 
-4. Click **Browse File to Upload** and select the relevant API from the `<APIM_HOME>/<OB_APIM_TOOLKIT_HOME>/repository/resources/apis` directory.
+4. Click **Browse File to Upload** and select the relevant API from the `<APIM_HOME>/<OB_APIM_TOOLKIT_HOME>/repository/resources/apis/consumerdatastandards.org.au/1.8.0/consumer-data-standards-1.8.0.yaml` directory.
 
 5. Click **Next**.
 
-6. Click **Create** to create the API. ![create-accounts](../assets/img/get-started/quick-start-guide/create-accounts.png)
+6. Set the **Endpoint** as follows:
+   ```
+   https://localhost:9443/api/openbanking/cds/backend/services
```

Comment:
since this is the Try Out section (not the QSG) let's use <IS_HOST> place holder instead of localhost here